packages/front-generator/src/common/model/cuba-model.ts

```diff
@@ -76,7 +76,8 @@ export interface EntityAttribute {
   column: string;
   mandatory: boolean;
   unique: boolean;
-  length: number;
+  length: number; // TODO VP: should be optional, but that would break polymer generator
```

Comment:
@web-devel may be it's time to stop polymer support?

Reply:
This will be discussed internally